my-app/backend/src/index.ts

```diff
@@ -0,0 +1,38 @@
+import cors from 'cors'
+import express,{Request,Response} from 'express'
+import * as mongoose from "mongoose";
+import {body, validationResult} from 'express-validator'
+
+
+mongoose
+    .connect('mongodb+srv://admin:admin123@cluster0.ttgwg1i.mongodb.net/?retryWrites=true&w=majority')
+    .then(() => console.log('DB OK'))
+    .catch((err) => console.log('DB ERROR', err));
+
+const app = express()
+const port = 8888
+app.use(cors())
```

Comment:
CORS is needed probably only in production, it is enabled to secure BE from accessing it from different domains. But for local development you might need to disable it. Just a note.

Reply:
ok